.github/workflows/source_aws.yml

```diff
@@ -54,7 +54,7 @@ jobs:
           cache: true
           cache-dependency-path: plugins/source/aws/go.sum
       - name: golangci-lint
-        uses: golangci/golangci-lint-action@v3
+        uses: bbernays/golangci-lint-action@58bce521b66ed125524e24eeec9522d765a8c2a7
```

Comment:
I would put the fork under `cloudquery/` and also use the branch name directly `cloudquery/golangci-lint-action@master` so if we follow up with fixes we don't have to update this `sha`